src/hlsBinaries.ts

```diff
@@ -217,36 +496,19 @@ async function getReleaseMetadata(
       path: releasesUrl.path,
     }
     : {
-      host: 'api.github.com',
-      path: '/repos/haskell/haskell-language-server/releases',
+      host: 'gist.githubusercontent.com',
+      path: '/hasufell/dd84df5f81a3a7e6e6fad8f122dba429/raw/73efc1078555d971076d3ccf31154f10ed683a82/hls-metadata.json',
```

Comment:
So this is obviously a temporary url. It contains the HLS-GHC mapping: https://gist.githubusercontent.com/hasufell/dd84df5f81a3a7e6e6fad8f122dba429/raw/73efc1078555d971076d3ccf31154f10ed683a82/hls-metadata.yaml

From this we figure out the latest HLS version that supports a given GHC.

The idea is that ghcup will generate this file on new releases, see https://github.com/haskell/ghcup-metadata/pull/22

Reply:
I was suggesting this should be hosted GHCup, but since this actually purely HLS metadata, we should host it somewhere assocaited with HLS... Maybe as a file in HLS directly?

Reply:
> I was suggesting this should be hosted GHCup, but since this actually purely HLS metadata, we should host it somewhere assocaited with HLS... Maybe as a file in HLS directly?

The metadata only changes when ghcup changes metadata, so I'd say it should be in ghcup-metadata repo.

Reply:
It is totally legitimate in the ghcup-metadata repo, so yes, let's go for it!